### ISSUE TITLE
Request -- Addition of vertical indent lines --

### ISSUE THREAD

**Issue by copobotics** [thonny/thonny#1078]
Hi there, Thonny!

 I have been using this program for a while now and I find amazing everything you have done, although I am usually missing a feature most IDE have, which is an option to allow vertical indentation lines. They would help making the code way clearer, specially when you have lots of nested loops and conditions. 

Thank you for making this possible and I hope we can all enjoy some pretty indent lines sooner than later ;)

**Comment by aivarannamaa** [comment]
Unfortunately I don't know how to do it in cross-platform way. I could use a trick with [bgstipple](https://www.tcl.tk/man/tcl8.6/TkCmd/text.htm#M45) in the Text widget, but it doesn't work in macOS.

**Comment by copobotics** [comment]
As long as it is implemented as optional for those who don't want it and doesn't take much of your time I'd be elated to see it, even if it's not available for mac users :)

**Comment by AlvaroHermida** [comment]
Yes, please, don't let Tim Cook's "management" hinder the development of Thonny.

**Comment by abhijit1990** [comment]
Is it done yet?

**Comment by aivarannamaa** [comment]
No, it is not yet done.

**Comment by yuanyanhui** [comment]
This is a good feature to have especially for beginners.

**Comment by superellips** [comment]
I agree with @yuanyanhui , would be good for beginners. I had a student looking for a feature like this when struggling with indentation during a lesson recently.

**Comment by abhijit1990** [comment]
Is it implemented?

**Comment by superellips** [comment]
No, we wished for it but couldn't find it.
